### ISSUE TITLE
Conflicting rules: block-opening-brace-space-before and media-query-list-comma-newline-after 

### ISSUE THREAD

**Issue by ntwb** [stylelint/stylelint#388]
Using the following CSS rules there appears to be conflicts between the `block-opening-brace-space-before` and `media-query-list-comma-newline-after` rules:

When using the following rules:
```
    "block-opening-brace-newline-after": [ 2, "always" ],
    "block-opening-brace-space-before": [ 2, "always" ],
    "media-query-list-comma-newline-after": [ 2, "always-multi-line" ],
    "media-query-list-comma-space-after": [ 2, "always-single-line" ],
    "media-query-list-comma-space-before": [ 2, "never" ],
```

Example 1: Given the following CSS:  (Single space between `) {` on line 1)

```css
@media only screen and (max-width: 640px), screen and (max-height: 400px) {
	.upload-php .mode-grid .media-sidebar{
		max-width: 100%;
	}
}
```

The following warning is displayed:
```
1187:38	⚠  Expected single space before "{" (block-opening-brace-space-before) [stylelint]
```

Example 2: Altering the CSS to: (Newline after the `,`)

```css
@media only screen and (max-width: 640px), 
screen and (max-height: 400px) {
	.upload-php .mode-grid .media-sidebar{
		max-width: 100%;
	}
}
```

The following warning is displayed:
```
1188:38	⚠  Expected single space before "{" (block-opening-brace-space-before) [stylelint]
```

Example 3: Altering the CSS to: (No whitespace between `){` on line 1)

```css
@media only screen and (max-width: 640px), screen and (max-height: 400px){
	.upload-php .mode-grid .media-sidebar{
		max-width: 100%;
	}
}
```

The following warning is displayed:
```
1186:73	⚠  Expected single space before "{" (block-opening-brace-space-before) [stylelint]
1187:38	⚠  Expected single space before "{" (block-opening-brace-space-before) [stylelint]
```

**Comment by davidtheclark** [comment]
Isn't the warning at `1187:38` about your rule, which is missing a space before its opening brace: `.upload-php .mode-grid .media-sidebar{`? Not about the media query opening brace?

If you add a space there — ``.upload-php .mode-grid .media-sidebar {` — does that fix the confusion?

If that's the case, I don't think the rule is malfunctioning.

Or am I missing something?

**Comment by ntwb** [comment]
> Or am I missing something?

Nope, right you are, this is what I get after going through ~32,000 lines of CSS 😳

**Comment by farhadniaz** [comment]
I have same issue for media queries  for example 
first time with --fix
` @media all and (max-width: 768px) `
===> 
` @media     all and (max-width: 768px) `

second time with --fix
` @media               all and (max-width: 768px) `

its add more space to styles after @mdeia

😳!!!!!

**Comment by jeddy3** [comment]
@farhadniaz Please create a new issue, using the "Report a bug" template, so we can better help you.

**Comment by farhadniaz** [comment]
@jeddy3  i will .
